<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>postnotifications</class>
 <widget class="QDialog" name="postnotifications">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>665</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Post Notice</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/new/img/Img/logo.jpg</normaloff>:/new/img/Img/logo.jpg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 89, 234);
</string>
  </property>
  <widget class="QLabel" name="label_notice">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>Notice :-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_submit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>600</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 0, 179);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>SEND</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>600</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 0, 179);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>CANCEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_gotohome">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>610</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(80, 161, 241);
border-radius:20%;


font: 75 16pt &quot;MS Shell Dlg 2&quot;;
font: 75 16pt &quot;Rockwell&quot;;</string>
   </property>
   <property name="text">
    <string>Go To Home</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_notice_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>From:-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_notice_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Trebuchet MS&quot;;</string>
   </property>
   <property name="text">
    <string>To:-</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_to">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_from">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>180</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>30</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/img/Img/logout.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gadugi</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="text">
    <string>POST NOTIFICATIONS </string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_notice">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>681</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="backgroundVisible">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string>Enter Your Data Here</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
